src/SLCore.UnitTests/Integration/SLCoreIntegrationSmokeTest.cs

```diff
@@ -55,8 +52,8 @@ public async Task SLCore_InitializeStandaloneAndShutdown()
 
         var slCoreServiceProvider = new SLCoreServiceProvider(new NoOpThreadHandler(), logger);
         slCoreServiceProvider.SetCurrentConnection(slCoreRunner.Rpc);
-        var slCoreListenerSetUp = new SLCoreListenerSetUp(new[] { new LoggerListener(new TestLogger(logToConsole: true)) });
```

Comment:
LoggerListener was not accessible so I commented out the line. Test is ignored anyway.

Reply:
Why not fix it? You can add internalsvisibleto easily

Reply:
it's going to be mostly rewritten in next sprint anyway.

Reply:
And you will still need that internalsvisibleto for that assembly 😒